<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameMainMenuWindow</class>
 <widget class="QWidget" name="GameMainMenuWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>camPAIN</string>
  </property>
  <property name="styleSheet">
   <string notr="true">     background-color:white;
</string>
  </property>
  <widget class="QPushButton" name="logoutButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>125</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	color:red;
	border: 1px solid #000000;
					 border-radius: 15px;
}

QPushButton:pressed
{
	color:red;
     background-color:rgba(0, 145, 255, 128);
}</string>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
  </widget>
  <widget class="QPushButton" name="characterBioButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>430</y>
     <width>200</width>
     <height>75</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	color:blue;
	border: 1px solid #000000;
					 border-radius: 15px;
}

QPushButton:pressed
{
	color:red;
     background-color:rgba(0, 145, 255, 128);
}</string>
   </property>
   <property name="text">
    <string>Character Bio</string>
   </property>
  </widget>
  <widget class="QPushButton" name="continueGameButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>230</y>
     <width>200</width>
     <height>75</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	border: 1px solid #000000;
					 border-radius: 15px;
}
</string>
   </property>
   <property name="text">
    <string>Continue Game</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startNewGameButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>330</y>
     <width>200</width>
     <height>75</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	color:blue;
	border: 1px solid #000000;
					 border-radius: 15px;
}

QPushButton:pressed
{
	color:red;
     background-color:rgba(0, 145, 255, 128);
}</string>
   </property>
   <property name="text">
    <string>Start New Game</string>
   </property>
  </widget>
  <widget class="QLabel" name="usernameLabel">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>60</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>username:</string>
   </property>
  </widget>
  <widget class="QLabel" name="totalVotesLabel">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>100</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>total votes:</string>
   </property>
  </widget>
  <widget class="QLabel" name="highScoreLabel">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>140</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>high score:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="campainLeaderboardButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>530</y>
     <width>200</width>
     <height>75</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	color:blue;
	border: 1px solid #000000;
					 border-radius: 15px;
}

QPushButton:pressed
{
	color:red;
     background-color:rgba(0, 145, 255, 128);
}</string>
   </property>
   <property name="text">
    <string>Leaderboard</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_34">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>60</y>
     <width>531</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DIN Condensed</family>
     <pointsize>100</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:red;
     background-color:rgba(255, 255, 255, 0);
</string>
   </property>
   <property name="text">
    <string>camPAIN - Menu</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Dokeylabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>290</y>
     <width>261</width>
     <height>211</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/head/Resources/elephant.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="elephantLabel">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>290</y>
     <width>261</width>
     <height>211</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/head/Resources/donkey.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
